demo-project/conf/base/catalog_08_reporting.yml

```diff
@@ -16,26 +16,25 @@ reporting.cancellation_policy_breakdown:
 
 reporting.price_histogram:
   type: plotly.JSONDataSet # Constructed via Python API
-  filepath: ${base_location}/08_reporting/price_histogram.json
+  filepath: ${_base_location}/08_reporting/price_histogram.json
   metadata:
     kedro-viz:
       layer: reporting
   versioned: true
 
 reporting.feature_importance:
   type: plotly.JSONDataSet # Constructed via Python API
-  filepath: ${base_location}/08_reporting/feature_importance_plot.json
+  filepath: ${_base_location}/08_reporting/feature_importance_plot.json
   metadata:
     kedro-viz:
       layer: reporting
   versioned: true
 
 reporting.cancellation_policy_grid:
   type: demo_project.extras.datasets.image_dataset.ImageDataSet
```

Comment:
I tried running the project locally with kedro-datasets 1.8.0 and it fails at inferring the type at - `demo_project.extras.datasets.image_dataset.ImageDataSet`. Should we modify it to `pillow.ImageDataSet` in this PR or take that up in other PRs ? @merelcht

Reply:
What's the error? That dataset is part of the `demo_project` so the rename in kedro-datasets shouldn't have any impact on this.

Reply:
Somehow the extras folder was missing earlier, cleaned it and it works fine now. Thank you Merel 💯